.github/workflows/create-gui-pr.yml

```diff
@@ -34,7 +34,7 @@ jobs:
   # Creates a pull request in the main application to update its GUI.
   create-gui-pr:
     # Only runs this job when the triggering workflow run was a success (i.e. the “Tests” workflow passes).
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    if: ${{ github.event_name == 'workflow_dispatch'  || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
```

Comment:
**Comment**: This check was incorrect and prevented the workflow from running when dispatching it manually. The condition would always be false then.

Reply:
Thanks for inline explanation 👍